subsys/bluetooth/controller/ll_sw/ull_llcp.h

```diff
@@ -54,6 +54,10 @@ struct ull_cp_conn {
 			uint8_t min_used_chans;
 		} muc;
 
+		/* Use by ACL Termination Procedure */
+		struct {
+			uint8_t error_code;
```

Comment:
Does the error_code need to be a persistent field? Isn't it only a transient value, as the link will be disconnected so there is no value in remembering it.

Reply:
after disconnect is acknowledged we need to send event to host with proper disconnect reason (also for locally initiated disconnect)